src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelListItem.tsx

```diff
@@ -75,13 +75,51 @@ export const TrackPanelListItem = (props: TrackPanelListItemProps) => {
     isDrawerOpened,
     drawerView,
     track,
-    trackStatus
+    trackStatus,
+    defaultTrackStatus
   } = props;
 
+  const updateTrackStates = (
+    trackId: string,
+    newStatus: TrackActivityStatus
+  ) => {
+    if (!activeGenomeId || !activeEnsObjectId) {
+      return;
+    }
+
+    // FIXME: Temporary hack until we have a set of proper track names
+    if (track.track_id.startsWith('track:gene')) {
```

Comment:
Ids of these tracks:

![image](https://user-images.githubusercontent.com/6834224/71363131-05bcd080-2590-11ea-859a-0e2e4e5ee560.png)

also start with track:gene (e.g. `track:gene-pc-fwd`), but belong in a different segment of redux state. Isn't it going to be a problem with this approach?